src/artm_tests/collection_parser_test.cc

```diff
@@ -132,19 +101,28 @@ TEST(CollectionParser, Multiclass) {
   ::artm::CollectionParserConfig config;
   config.set_format(::artm::CollectionParserConfig_Format_BagOfWordsUci);
   config.set_target_folder(target_folder);
-  config.set_dictionary_file_name("test_parser.dictionary");
   config.set_vocab_file_path("../../../test_data/vocab.parser_test_multiclass.txt");
   config.set_docword_file_path("../../../test_data/docword.parser_test.txt");
 
-  std::shared_ptr< ::artm::DictionaryConfig> dictionary_parsed = ::artm::ParseCollection(config);
```

Comment:
Same question here as for the test above (` TEST(CollectionParser, UciBagOfWords)`).
Let's verify that `ArtmGatherDictionary` is able to handle multi-class vocab files.